### ISSUE TITLE
structure fine-tuning  issue

### ISSUE THREAD

**Issue by tzktok** [microsoft/table-transformer#181]
Hi, I am trying to fine tune the the table structure over on TATR-v1.1-All-msft.pth model...In my dataset i provide an 50 images for finetune...i am getting below training metrics...
```
(tables-detr) PS E:\table-transformer\src> python main.py --data_type structure --config_file structure_config.json --data_root_dir data_set --model_load_path TATR-v1.1-All-msft.pth --epochs 50  --mode train --model_save_dir fina
l_model --load_weights_only

```
```
pubmed: AP50: 0.811, AP75: 0.637, AP: 0.602, AR: 0.713
Total training time:  0:02:20.607232
```
How to increase the training accuracy more...i am using default structure config..In dataset how many images needed to get more better result..

@bsmock   @dnfclas

**Comment by YingxuanW** [comment]
I just saw someone use 1k datas to fine-tuning the detection model.

**Comment by YingxuanW** [comment]
> Hi, I am trying to fine tune the the table structure over on TATR-v1.1-All-msft.pth model...In my dataset i provide an 50 images for finetune...i am getting below training metrics...
> 
> ```
> (tables-detr) PS E:\table-transformer\src> python main.py --data_type structure --config_file structure_config.json --data_root_dir data_set --model_load_path TATR-v1.1-All-msft.pth --epochs 50  --mode train --model_save_dir fina
> l_model --load_weights_only
> ```
> 
> ```
> pubmed: AP50: 0.811, AP75: 0.637, AP: 0.602, AR: 0.713
> Total training time:  0:02:20.607232
> ```
> 
> How to increase the training accuracy more...i am using default structure config..In dataset how many images needed to get more better result..
> 
> @bsmock @dnfclas

Can you please tell me how did you prepare your fine-tuning datas? I also need to fine-tune thee table-structure model!

**Comment by dreamlychina** [comment]
> > Hi, I am trying to fine tune the the table structure over on TATR-v1.1-All-msft.pth model...In my dataset i provide an 50 images for finetune...i am getting below training metrics...
> > ```
> > (tables-detr) PS E:\table-transformer\src> python main.py --data_type structure --config_file structure_config.json --data_root_dir data_set --model_load_path TATR-v1.1-All-msft.pth --epochs 50  --mode train --model_save_dir fina
> > l_model --load_weights_only
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ```
> > pubmed: AP50: 0.811, AP75: 0.637, AP: 0.602, AR: 0.713
> > Total training time:  0:02:20.607232
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > How to increase the training accuracy more...i am using default structure config..In dataset how many images needed to get more better result..
> > [@bsmock](https://github.com/bsmock) [@dnfclas](https://github.com/dnfclas)
> 
> Can you please tell me how did you prepare your fine-tuning datas? I also need to fine-tune thee table-structure model!

大佬你能微调？交流下
